lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -266,6 +266,9 @@ protected function transformCampaign($data) {
         $output['uri'] = $data->uri;
       }
 
+      if (dosomething_helpers_get_variable('node', $data->id, 'magic_link_copy')) {
```

Comment:
@angaither `dosomething_helpers_get_variable('node', $data->id, 'magic_link_copy')` is either `FALSE` or the `magic_link_copy` text. So in the new commit it either returns the `magic_link_copy` value or leaves out that attribute. Lemme know if I used too many pronouns again haha